options/locale/locale_ru-RU.ini

```diff
@@ -373,6 +373,7 @@ old_password=Текущий пароль
 new_password=Новый пароль
 retype_new_password=Подтверждение нового пароля
 password_incorrect=Текущий пароль неправильный.
+password_complexity =  Пароль простой.
```

Comment:
You cannot change this file from here... Only English translations are allowed to be added via a PR.
You will need to make use of [Crowdin](http://crowdin.com) for this.

Reply:
You will have to delete this changes and add it from Crowdin